mapstruct-lombok/build.gradle

```diff
@@ -20,7 +19,8 @@ ext {
 sourceCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
-    compile "org.mapstruct:mapstruct-jdk8:${mapstructVersion}", "org.projectlombok:lombok:${lombokVersion}"
     testCompile 'junit:junit:4.12'
-    apt "org.mapstruct:mapstruct-processor:${mapstructVersion}", "org.projectlombok:lombok:${lombokVersion}"
+    compile "org.mapstruct:mapstruct-jdk8:${mapstructVersion}"
+    compileOnly "org.mapstruct:mapstruct-processor:${mapstructVersion}"
```

Comment:
The reason why I liked the original set-up better is that it didn't expose the processor JAR on the compilation classpath at all. That way no code from the processor could accidentally be used within the compiled project.